spyder/plugins/outlineexplorer/tests/test_outline_explorer.py

```diff
@@ -30,6 +30,16 @@
     [63, 'if __name__ == "__main__":', 0, 2, 'if __name__ == "__main__":']
 ]
 
+oe_data_filtered = [oe_data[idx] for idx in [0, 1, 2, 3, 4, 6, 7, 8, 9]]
```

Comment:
Sorry, missed seeing it wasn't in the list. A clearer and more robust way of doing this, then, is

```suggestion
oe_data_filtered = [oe for oe in oe_data if oe[1][:3] not in ["if ", "for"]]
```

or, shorter and simpler,

```suggestion
oe_data_filtered = oe_data[:5] + oe_data[6:10]
```